tests/phpunit/unit-tests/Helper/Fields/Test_Field_Repeater.php

```diff
@@ -80,4 +80,20 @@ public function test_repeater_add_entry_meta_method() {
 		$this->assertSame( '172.17.0.1', $values[7] ); // These test expects integer results.
 		$this->assertSame( 'USD', $values[10] );
 	}
+
+	public function test_get_repeater_form_data() {
+		/* This method expects the third parameter as an GF_Field_Repeater Instance. */
+		$data          = $this->pdf_field->get_repeater_form_data( $this->pdf_field->form_data(), $this->pdf_field->value(), new \GF_Field_Repeater( $this->form['fields'][1] ) );
+		$repeater_data = $data['repeater']['999.Team'][0];
+
+		/* Verify that method correctly adds the HTML field with each corresponding indexes. */
+		$this->assertArrayHasKey( 17, $repeater_data );
```

Comment:
You should check the content of each of these three array keys to verify it is what you expect it to be.